extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -756,36 +778,28 @@ export class AmpLightboxGallery extends AMP.BaseElement {
           transformOrigin: 'top left',
           willChange: 'transform',
         });
-        transLayer.appendChild(clone);
-
-        // Move and resize the image to the location given by the lightbox.
         const dx = imageBox.left - rect.left;
         const dy = imageBox.top - rect.top;
         const scaleX = rect.width != 0 ? imageBox.width / rect.width : 1;
-
         duration = this.getTransitionDuration_(dy);
 
-        anim.add(MOTION_DURATION_RATIO - 0.01,
-            tr.setStyles(dev().assertElement(this.carousel_), {
-              opacity: tr.numeric(0, 1),
-            }),
-            0.01
-        );
-
+        // Animate the position and scale of the transition image to its
+        // final lightbox destination in the middle of the page
         anim.add(0, tr.setStyles(clone, {
           transform: tr.concat([
             tr.translate(tr.numeric(0, dx), tr.numeric(0, dy)),
             tr.scale(tr.numeric(1, scaleX)),
           ]),
         }), MOTION_DURATION_RATIO, ENTER_CURVE_);
-
-        // At the end, fade out the transition image.
-        anim.add(0.9, tr.setStyles(transLayer, {
-          opacity: tr.numeric(1, 0.01),
-        }), 0.1, EXIT_CURVE_);
-      } else {
-        st.setStyles(dev().assertElement(this.carousel_), {opacity: ''});
-      }
+      },
```

Comment:
The main point of this PR is to make sure that all mutations go in a `vsync_.mutate` instead of `vsync_.measure` as they were originally in. This adaptation was made in 2 places, respectively `enter_()` and `exit_()`.